### ISSUE TITLE
Set MUC Password only as fallback

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#56]
```
Did some testing with my muc server which handles the roomconfig_roomowners 
form just fine. So I though it would be nice to have an membersonly room when 
this setting works and as fallback a password protected room.

Here is the patch. Seems to work nice.

```

Original issue reported on code.google.com by `fschm...@gmail.com` on 20 Jan 2011 at 6:59

Attachments:
- [muc_password_as_fallback.patch](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-56/comment-0/muc_password_as_fallback.patch)

**Comment by GoogleCodeExporter** [comment]
```
The addition of "return null" in XmppManager.java after Log.e(Tools.LOG_TAG, 
"createRoom: room = " + room, ex);  is misleading/wrong, as this catch block 
catches the following exception (at least on jwchat.org): 
E/gtalksms(20127): Error on creating room: room = xx (00491xx)
E/gtalksms(20127): Creation failed - Missing acknowledge of room creation.: 
E/gtalksms(20127):   at 
org.jivesoftware.smackx.muc.MultiUserChat.create(MultiUserChat.java:343)
E/gtalksms(20127):   at 
com.googlecode.gtalksms.XmppManager.createRoom(XmppManager.java:422)

but the room is still created. 

```

Original comment by `fschm...@gmail.com` on 21 Jan 2011 at 11:06

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 21 Jan 2011 at 7:17
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 4:12
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.7.3
```

Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 8:07
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
